azext_edge/edge/providers/rpsaas/adr2/command.md

```diff
@@ -0,0 +1,476 @@
+
+### asset endpoint
+
+  create
+
+	--endpoint-profile [or --name]
+	--instance
+	--resource-group
+	--target-address
+	[optional]
+	--location
+	--instance-resource-group
+	--instance-subscription
+	--additional-configuration
+	--profile-type
+	--certificate-ref [use this, user/password-ref to determine auth-mode]
+	--password-ref
+	--user-ref
+	--tags
+
+  delete
+
+	--endpoint-profile [or --name]
+	--resource-group
+
+  list/query
+
+	[optional]
+	--discovered [bool flag]
+	--resource-group
+	--instance
+	--instance-resource-group [required if instance is used]
+	--target-address
+	--auth-mode
+	--profile-type
+	--resource-query
+
+  show
+
+	--endpoint-profile [or --name]
+	--resource-group
+
+  update
+
+	--endpoint-profile [or --name]
+	--resource-group
+	[optional]
+	--target-address
+	--additional-configuration
+	--profile-type
+	--certificate-ref [use this, user/password-ref to determine auth-mode]
+	--password-ref
+	--user-ref
+	--tags
+
+## Schemas
+
+Need to create schema registry first. Schema is a subresource. Schema version is a subresource of schema.
+
+In the end, the id (for version) looks like this: `/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.DeviceRegistry/schemaRegistries/my-schema-registry/schemas/my-schema/schemaVersions/1`
+
+custom location is not needed...
+
+### schema
+  create
+
+	--schema [or --name]
+	--schema-registry
+	--resource-group
+	--format
+	--schema-type
+	[optional]
+	--description
+	--display-name
+	--tags
+
+  delete
+
+	--schema [or --name]
+	--resource-group
+
+  show
+
+	--schema [or --name]
+	--resource-group
+
+  list
+
+	--resource-group
+	--schema-registry
+	[optional]
+	--format
+	--schema-type
+	--display-name
+
+### schema registry
+  create
+
+	--schema-registry [or --name]
+	--resource-group
+	--namespace
+	--storage-container-uri
+	[optional]
+	--display-name
+	--description
+	--tags
+
+  delete
+
+	--schema-registry [or --name]
+	--resource-group
+
+  show
+
+	--schema-registry [or --name]
+	--resource-group
+
+  list/query
+
+	[optional]
+	--resource-group
+	--display-name
+	--namespace
+	--storage-container-uri
+	--resource-query
+
+  update
+
+	--schema-registry [or --name]
+	--resource-group
+	[optional]
+	--display-name
+	--description
+	--tags
+
+### schema version
+  create
+
+	--schema-version [or --name]
+	--schema
+	--schema-registry
+	--resource-group
+	--content
+	[optional]
+	--description
+
+  delete
+
+	--schema-version [or --name]
+	--schema
+	--schema-registry
+	--resource-group
+
+  list
+
+	--schema
+	--schema-registry
+	--resource-group
+
+  show
+
+	--schema-version [or --name]
+	--schema
+	--schema-registry
+	--resource-group
+
+
+## Billing containers?
+### billing-containers [may prepend device-registry or asset to avoid confusion or just skip this]
```

Comment:
Purpose check

Reply:
idk shits and giggles? 

I cannot really tell the purpose until I play with it but I can see some users wanting billing info. At the same time - it would be the lowest priority in getting implemented